src/test/java/org/cactoos/text/TextOfTest.java

```diff
@@ -92,6 +92,22 @@ public void readsInputIntoTextWithSmallBuffer() throws Exception {
         );
     }
 
+    @Test
+    public void readsReaderIntoTextWithSmallBuffer() throws Exception {
+        MatcherAssert.assertThat(
+            "Can't read text from Reader with a small reading buffer",
+            new TextOf(
+                new StringReader("Hi there! with small buffer"),
+                2,
+                StandardCharsets.UTF_8
+            ).asString(),
+            Matchers.allOf(
+                Matchers.startsWith("Hi"),
+                Matchers.endsWith(" there! with small buffer")
+            )
```

Comment:
@krzyk We should avoid using `Matchers` static methods; try to use cactoos `Matchers` or `StringStartsWith`  and`StringEndsWith`.